packages/db/index.ts

```diff
@@ -1,13 +1,16 @@
 import { PrismaClient } from "@prisma/client";
 import { withAccelerate } from "@prisma/extension-accelerate";
 
-export const extendedPrisma = new PrismaClient({
-  log:
-    process.env.NODE_ENV === "development"
-      ? ["query", "error", "warn"]
-      : ["error"],
-}).$extends(withAccelerate());
+const createPrismaClient = () =>
```

Comment:
Bug - the singleton export logic here wasn't right so we were ending up with multiple Prisma clients